features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/fr.json

```diff
@@ -654,7 +657,23 @@
   "Apis.Details.Resources.components.AddOperation.clear.inputs.tooltip": "",
   "Apis.Details.Resources.components.AddParameter.add.tooltip": "",
   "Apis.Details.Resources.components.AddParameter.clear.inputs.tooltip": "",
+  "Apis.Details.Resources.components.operationComponents.\n                                OperationGovernance.operation.scope.create.new.scope": "",
```

Comment:
Shall we remove these new line in the key? Using string concatenation will help to do that.

Reply:
Sure

Reply:
They have been removed as requested